Library/Formula/crystal.rb

```diff
@@ -0,0 +1,22 @@
+class Crystal < Formula
+  desc "Code generator for every language, library and framework"
+  homepage "http://crystal.sh"
+  url "https://github.com/crystal/crystal/archive/v0.5.1.tar.gz"
+  sha256 "6c022ad59a10cec58856dba858a030b745333909108c08bd326c20ac0cb144fb"
+
+  depends_on "node"
+
+  def install
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  def post_install
+    cd libexec
+    system "npm", "install"
```

Comment:
This install Node.js dependencies. It depends on `node` which includes `npm`, but it doesn't seem to be accessible until `post_install`. Please let me know if I should be doing something differently. Thanks for your help!